LeanKit-Labs/autohost
# Updated project dependencies and security issues

Replaced biggulp with npm scripts
updated to eslint, nyc
specified Node v 12.18
Updated packages:
- express
- fount
- lodash
- request
- socket.io
- whistlepunk

removed all Metrics gathering

TESTING INSTRUCTIONS:
Link in autohost to any projects that use it.  Run the tests and test endpoints to make sure everything still functions.
